demos/About Dialog/code.rs

```diff
@@ -41,5 +40,5 @@ fn on_button_clicked() {
 
     dialog.add_acknowledgement_section(Some("Special thanks to"), &["My cat"]);
 
-    dialog.present();
+    dialog.present(workbench::window);
```

Comment:
```suggestion
    dialog.present(workbench::window());
```

I am pretty sure it must be this. Did you test this example? :)